skyrim-platform/src/platform_se/skyrim_platform/EventHandler.cpp

```diff
@@ -688,20 +801,25 @@ EventResult EventHandler::ProcessEvent(
     return EventResult::kContinue;
   }
 
-  auto e = CopyEventPtr(event);
+  auto activeRefFormId = event->activeRef.get()->GetFormID();
```

Comment:
event->activeRef.get() may be nullptr. needs check here

Use smth like event->subject ? event->subject->GetFormID() : 0;